docs/discourse.adoc

```diff
@@ -0,0 +1,75 @@
+= Discourse
+
+This document explains how the FOSS@MAGIC Discourse site, https://fossrit.community[fossrit.community], is set up.
+It is divided into the following sections:
+
+. What is Discourse?
+. Hosting: DigitalOcean
+. Configuration: Docker-based deployment
+. Updates: How to update Discourse
+. Backups: Discourse and DigitalOcean
+
+
+== What is Discourse?
+
+https://www.discourse.org/[Discourse] is open source forum software developed by Civilized Discourse Construction Kit, Inc.
+In Fall 2018, FOSS@MAGIC student employee and RIT student https://twitter.com/jflory7[Justin W. Flory] set up a Discourse site for the FOSS@MAGIC community on https://fossrit.community[fossrit.community].
+This decision was approved by the RIT MAGIC Center staff.
+Since then, we use Discourse as a place for our community to congregate from around the world.
+It intends to be a hub of information and a place for discussion within the RIT FOSS community.
+
+
+[hosting]
+== Hosting: DigitalOcean
+
+The Discourse site is hosted on a https://www.digitalocean.com/[DigitalOcean] Droplet.
+The Droplet is managed through the _FOSS at RIT MAGIC_ team on DigitalOcean.
+The following individuals have access to the DigitalOcean team:
```

Comment:
Should we state this? Seems a bit unnecessary and potentially a security risk.

Reply:
@Nolski I took inspiration from the expansive [Fedora Infrastructure SOP documentation](https://docs.pagure.org/infra-docs/sysadmin-guide/index.html#standard-operating-procedures). I think it's worth noting who has access and what degree of access they have since it creates a record of who the points-of-contact are for gaining infrastructure access. In two years after I graduate, there should be a way for future FOSS@MAGIC payroll students to get on-boarded to the infrastructure and who to contact for that to happen.

Reply:
Definitely up to you but I would suggest maybe just having a single point of contact (like a webmaster or whatever) instead of exposing users to be targets of phishing due to their roles being public.

Reply:
@Nolski I appreciate your feedback and get where you're coming from. I think it is still worth including for when someone, some day, needs to figure out how to gain access. I may not be as accessible as I am now since I have one semester left. Since the only other person with access to the Droplet today is someone who is not normally involved with FOSSRIT infrastructure, I think it's worth documenting.